notes/2021-01-07.md

```diff
@@ -0,0 +1,213 @@
+# GraphQL WG Notes – January 2021
+
+**Watch the replay:** [GraphQL Working Group Meetings on YouTube](https://www.youtube.com/playlist?list=PLP1igyLx8foH30_sDnEZnxV_8pYW3SDtb)
+
+1. Agree to Membership Agreement, Participation Guidelines and Code of Conduct (1m, Lee)
+   - [Specification Membership Agreement](https://github.com/graphql/foundation)
+   - [Participation Guidelines](https://github.com/graphql/graphql-wg#participation-guidelines)
+   - [Code of Conduct](https://github.com/graphql/foundation/blob/master/CODE-OF-CONDUCT.md)
+2. Introduction of attendees (5m, Lee)
+3. Determine volunteers for note taking (1m, Lee)
+4. Review agenda (2m, Lee)
+5. Review previous meeting's action items (5m, Lee)
+   - [All action items](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+label%3A%22Action+item+%3Aclapper%3A%22+sort%3Aupdated-desc)
+   - Input Unions WG 7pm UTC (11am PST, 6am AEDT) on Thursday 21st Jan ([Details in Slack](https://graphql.slack.com/archives/CTECCMU57/p1608223711031900?thread_ts=1606562127.018400&cid=CTECCMU57))
```

Comment:
January 7 ?

Reply:
We had the input unions WG yesterday (Jan 21st)